src/goto-instrument/remove_function.cpp

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************\
+
+Module: Remove function definition
+
+Author: Michael Tautschnig
+
+Date: April 2017
```

Comment:
This comment block doesn't normally have the date, how come we've added it here?